t/data/40-templates.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
This is not a JSON document. I'd rather call the file `40-templates.pl`.

Reply:
You're right. I was planning to add both and mixed it up in the process :sweat_smile: